SimG4CMS/HcalTestBeam/test/python/run2006_cfg.py

```diff
@@ -109,6 +109,8 @@
     MaxTrackTime  = cms.double(1000.0),
     MaxTimeNames  = cms.vstring(),
     MaxTrackTimes = cms.vdouble(),
+    MaxTrackTimeForward = cms.double(2000.0), # ns
```

Comment:
@bsunanda , as far as I remember the max time in TB 2006 was 1000 ns, also why you need to change MaxZCentralCMS?

Reply:
The common_maximum_timex is used in the local definition of PS SteepingAction and CaloSD. Running the code complained about missing parameter MaxTrackTimeForward and MaxZCentralCMS - so I copied the ones from default g4SimHits there